web/crux/src/services/domain.notification.service.ts

```diff
@@ -49,4 +61,19 @@ export class DomainNotificationService {
       this.logger.error(err)
     }
   }
+
+  private getMessageEventFromType(messageType: NotificationMessageType): NotificationEventTypeEnum {
+    switch (messageType) {
+      case 'node':
+        return NotificationEventTypeEnum.nodeAdded
+      case 'version':
+        return NotificationEventTypeEnum.versionCreated
+      case 'invite':
+        return NotificationEventTypeEnum.userInvited
+      case 'failedDeploy':
+        return NotificationEventTypeEnum.deploymentCreated
+      case 'successfulDeploy':
+        return NotificationEventTypeEnum.deploymentCreated
+    }
```

Comment:
there is a mapper for this in the mapper class?

Reply:
I don't think so. This maps the template type to the notification event type.

Reply:
```
add default branch pls
```

Reply:
It's already added :)